cmd/istioctl/main.go

```diff
@@ -147,14 +155,20 @@ and destination policies.
 					}
 				}
 
-				newRev, err := configClient.Update(config)
-				if err != nil {
+				var newRev string
+				if newRev, err = configClient.Update(config); err != nil {
 					return err
 				}
 
 				fmt.Printf("Updated config %v to revision %v\n", config.Key(), newRev)
 			}
 
+			if len(others) > 0 {
+				if err = runKubectl("replace", others); err != nil {
+					return err
```

Comment:
why do we need this? isnt crd generic enough that we can just submit it to api server? This is like writing shell script in Go. The whole idea behind istioctl was to expose the minimal required api server functionality to end user. Requiring end users to install kubectl and yet use istioctl (which just execs kubectl) seems counter intuitive.